apps/client/src/app/components/home-holdings/home-holdings.component.ts

```diff
@@ -119,6 +119,14 @@ export class HomeHoldingsComponent implements OnDestroy, OnInit {
     }
   }
 
+  public onHoldingClicked({ dataSource, symbol }: AssetProfileIdentifier) {
```

Comment:
Please move the method up that the methods are sorted alphabetically.

Reply:
> Please move the method up that the methods are sorted alphabetically.

Hey @dtslvr, Thanks for the feedback. Can you please clarify where exactly it should be present, Should I place the method below onChangeHoldingType() method?

Reply:
Yes, exactly 👍🏻